source/client/model/ModelPart.cpp

```diff
@@ -173,38 +173,6 @@ void ModelPart::render(float scale)
 	}
 }
 
-void ModelPart::renderHorrible(float scale)
```

Comment:
First, it's called `renderHorrible`, it's obviously not particularly *good*. Second, it was only used by the Camera entity, and changing it to use `render` makes no difference. Also, we've already removed `renderHorrible` from classes (maybe Cube?) in the past. I've kept `renderRollable` in, because I have no idea what it does.

This was just me cleaning up code that was not used, nor should it be according to the function name.